src/foam/box/MessagePortBox.js

```diff
@@ -27,25 +27,33 @@ foam.CLASS({
     'foam.box.Message'
   ],
 
-  imports: [ 'messagePortService', 'me' ],
+  imports: [
+    'generator? as ctxGenerator',
```

Comment:
Same question, also I'm not a fan of the name "generator" in the context.  It's a serializer or marshaller.  Generator has other connotations in comp sci.